server/graphql/resolvers.ts

```diff
@@ -27,6 +28,52 @@ const getRequestsById = (requestIds, dataSources) => {
   return requestIds.map(id => dataSources.requests.getById(id))
 }
 
+const updateRequestGroupHelper = (requestGroup, dataSources, dateUpdated = Date.now()): Promise<ServerResponseInterface> => {
+  requestGroup.dateUpdated = dateUpdated;
```

Comment:
I didn't put the check here since we don't use requestGroup.id here. There is still the check in the common update() method in cachedMongooseDataSource so I left it out here.

Reply:
icic makes sense!
lol i totally missed that